README.md

```diff
@@ -177,17 +164,17 @@ NAME                                     READY   STATUS      RESTARTS   AGE
 kogito-cloud-operator-6d7b6d4466-9ng8t   1/1     Running     0          26m
 ```
 
-Use the pod name as the input of the following command:
+Use the pod name as the input in the following command:
 
 ```bash
 $ oc logs -f kogito-cloud-operator-6d7b6d4466-9ng8t
 ```
 
-#### Native build fails with "The build pod was killed due to an out of memory condition."
+#### Native build fails with "The build pod was killed due to an out of memory condition." error
 
-By default, the operator will set the limit in the s2i build pod to 10GB of memory and 1 CPU. It might happen that your Kogito service would require even more resources from the cluster. 
+By default, the Operator sets the limit in the S2I build pod to 10GB of memory and 1 CPU. Your Kogito service might require more resources from the cluster.
```

Comment:
@ricardozanini This should be deleted has we don't have build limits anymore in 0.6.0 ?

Reply:
@ricardozanini , please confirm.

Reply:
@radtriste correct

Reply:
Deleted that section on builds failing due to insufficient build limit.

Reply:
Don't forget also to update the TOC :)